version_test.go

```diff
@@ -0,0 +1,14 @@
+package specs
+
+import (
+	"regexp"
+	"testing"
+)
+
+func TestVersion(t *testing.T) {
+	re := regexp.MustCompile(`\d+\.\d+\.\d+`)
```

Comment:
On Thu, Sep 10, 2015 at 12:33:20PM -0700, Vincent Batts wrote:
> > +	re := regexp.MustCompile(`\d+\.\d+\.\d+`)
> 
> This _test.go can be dropped entirely…

That's what I'd recommend.